test/login/components/SearchHelpSignIn.unit.spec.jsx

```diff
@@ -4,18 +4,29 @@ import { expect } from 'chai';
 
 import { SearchHelpSignIn } from '../../../src/js/login/components/SearchHelpSignIn.jsx';
 
+const defaultLoginProps = {
```

Comment:
I had to refactor this unit test in order to keep it DRY while adding the one test to cover the `null` greeting fix.